definitelyTypedTests/src/transformer/transformer.ts

```diff
@@ -0,0 +1,66 @@
+import { TypeChecker } from '../../../src/transformer/typeChecker/typeChecker';
+import { getMock } from '../../../src/transformer/mock/mock';
+import {
+    CustomFunction
+} from "../../../src/transformer/matcher/matcher";
+import * as ts from 'typescript';
+import { GetProgram } from "../../../src/transformer/program/program";
+import { baseTransformer } from '../../../src/transformer/base/base';
+import { GetTypeQueryDescriptor } from '../../../src/transformer/descriptor/typeQuery/typeQuery';
+import { Scope } from '../../../src/transformer/scope/scope';
+import { DefinitelyTypedTransformerLogger } from './logger';
+import * as path from 'path';
+
+const customFunctions: CustomFunction[] = [
+    {
+        sourceDts: 'create-definitely-typed-mock.d.ts',
+        sourceUrl: '../create-definitely-typed-mock.d.ts',
+    }
+];
+
+const transformer = baseTransformer(visitNode, customFunctions);
+export { transformer };
+
+function visitNode(node: ts.CallExpression, declaration: ts.FunctionDeclaration): ts.Node {
+    const typeQueryNode: ts.TypeNode = node.typeArguments[0];
+
+    if (isCreateDefinitelyTypedMock(declaration) && ts.isTypeQueryNode(typeQueryNode)) {
+        const typeChecker: ts.TypeChecker = TypeChecker();
+        const typeQuerySymbol: ts.Symbol = typeChecker.getSymbolAtLocation(typeQueryNode.exprName);
+        const typeQuerySymbolDeclaration: ts.ImportEqualsDeclaration = typeQuerySymbol.declarations[0] as ts.ImportEqualsDeclaration;
+        const symbolAlias: ts.Symbol = typeChecker.getSymbolAtLocation(typeQuerySymbolDeclaration.name);
+        const symbol: ts.Symbol = typeChecker.getAliasedSymbol(symbolAlias);
+
+        if (!symbol.declarations) {
+            const moduleName: string =
```

Comment:
Basically this is the scenario when "is not a module".
 It tries to resolve valid declarations from the module and it will create a mock for each.